interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -402,7 +381,8 @@ namespace cling {
       m_MetaProcessor.getOuts() << (*I).first->getName();
       m_MetaProcessor.getOuts() << "\n";
     }
-    /* Only available in clang's trunk:
+#if 0
```

Comment:
We can probably enable this now? I guess "clang's trunk" refers to clang from 5 years ago :-) Can you give it a try? (Subsequent PR is fine.)

Reply:
Yes, of course!  Will do on a different PR, though.